axelrod/tests/strategies/test_player.py

```diff
@@ -359,9 +359,6 @@ def test_responses(test_class, player1, player2, responses, history1=None,
     for response in responses:
         s1, s2 = simulate_play(player1, player2)
         test_class.assertEqual(s1, response)
-    if attrs:
```

Comment:
Why do we not need `attrs` anymore? Have all such cases been moved to `versus_test`?